security/openconnect/src/opnsense/mvc/app/models/OPNsense/Openconnect/General.xml

```diff
@@ -47,6 +47,18 @@
             <Type>cert</Type>
             <Required>N</Required>
         </clientcertificate>
+        <tokenmode type="OptionField">
+            <Required>N</Required>
+            <OptionValues>
+                <rsa>RSA SecurID</rsa>
```

Comment:
We may need a "none" or "off" here as well which is the old default. It doesn't seem to be possible to go back to that situation afterwards although one could argue the side effect of leaving the secret empty can do the trick it's not very straight forward and maybe you don't want the secret blanked for other reasons.

Reply:
Seems like since the field isn't required, there already is a 'none' option when using OptionField?

![image](https://user-images.githubusercontent.com/2361852/168220060-438a468d-62b9-4be5-839f-20a5dd93240e.png)

Reply:
Sounds good, sorry for the noise then :)